crypto/conf/conf_sap.c

```diff
@@ -39,10 +39,24 @@ void OPENSSL_config(const char *appname)
 }
 #endif
 
-void openssl_config_int(const char *appname)
+int openssl_config_int(const OPENSSL_INIT_SETTINGS *settings)
 {
+     int ret;
+     const char *filename;
+     const char *appname;
+     unsigned long flags;
+
     if (openssl_configured)
-        return;
+        return 1;
+
+     filename = settings ? settings->filename : NULL;
+     appname = settings ? settings->appname : NULL;
+     flags = settings ? settings->flags : DEFAULT_CONF_MFLAGS;
```

Comment:
and this ?

Reply:
Yes, good catch, thanks. I have a pending commit that fixes the indents, but I am not pushing it yet, no point burning CI cycles on whitespace changes.  Correct indentation will be in the final version.